src/app.module.ts

```
@@ -141,7 +141,7 @@ import { InAppNotificationAdminModule } from './platform-admin/in-app-notificati
           infer: true,
         });
         const dbType = dbOptions.type || 'postgres';
-
+        
```

ESLint

[error] Delete `········`

(prettier/prettier)